atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -1214,6 +1215,21 @@ CfDef getCfForTable(TableReference tableRef, byte[] rawMetadata, int gcGraceSeco
                 config);
     }
 
+    @Override
+    public List<byte[]> getRowKeysInRange(TableReference tableRef, byte[] startRowInclusive, int batchSize) {
+        RowGetter rowGetter = new RowGetter(clientPool, queryRunner, ConsistencyLevel.QUORUM, tableRef);
+        GetCellTimestamps getCellTimestamps = new GetCellTimestamps(
+                newInstrumentedCqlExecutor(),
+                rowGetter,
+                tableRef,
+                startRowInclusive,
+                batchSize,
+                config);
```

Comment:
It's an unfortunate consequence of coupling between sweep and `GetCellTimestamps`, but internally this has a `rowGetter.getRows("getCandidateCellsForSweeping", ...)` call which will likely be confusing e.g. for jstacks or the kvs-slow-log, as that name is used to identify what the KVS is trying to do.